osu.Desktop/Overlays/VersionManager.cs

```diff
@@ -91,6 +99,44 @@ private void load(NotificationOverlay notification, OsuColour colours, TextureSt
                 checkForUpdateAsync();
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            State = Visibility.Visible;
+
+            var version = game.Version;
+            var lastVersion = config.Get<string>(OsuSetting.Version);
+            if (char.IsNumber(version[0]) && version != lastVersion)
+            {
+                config.Set(OsuSetting.Version, version);
+
+                // only show a notification if we've previously saved a version to the config file (ie. not the first run).
+                if (!string.IsNullOrEmpty(lastVersion))

```

Comment:
can't do; the set to config needs to only happen under the conditionals it is contained by